.github/workflows/main.yml

```diff
@@ -33,39 +33,29 @@ jobs:
     #
     #
     # Build
-    - run: ./scripts/build.sh
+    - name: Build jar files
+      run: ./scripts/build.sh
 
     #
     #
     # Unit test
-    - run: ./test/test.sh unit.py
+    - name: Run unit tests
+      run: ./test/test.sh unit.py
 
     #
     #
     # Component test
-    - run: ./test/test.sh component.py
+    - name: Run component tests
+      run: ./test/test.sh component.py
 
     #
     #
     # Upload coverage results to coveralls
     - name: Submit Coveralls
```

Comment:
Please add condition to not execute this step on external PRs.

Reply:
Done